### PR TITLE
feat: Init FE repository using cri-templates #5

### PR DESCRIPTION
**Pull request by merlinc**
### What changed

- Add Express app & config
- Add hmpo-form-wizard config
- Add i18n locales
- Add Cucumber config
- Add basic "intro" page
- Add related browser tests

### Issue tracking

- [OJ-1788](https://govukverify.atlassian.net/browse/OJ-1788)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[OJ-1788]: https://govukverify.atlassian.net/browse/OJ-1788?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jkunle** [comment]
LGTM

**Review by jkunle — APPROVED** [review]

**Comment by scronk1** [comment]
@merlinc Can we rename the PR? 20K line changes doesn't feel like a chore

File: tests/browser/pages/relying-party.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    this.startingUrl = `${websiteHost}/oauth2/authorize?request=lorem&client_id=${this.clientId}`;
+
+    await this.page.goto(this.startingUrl);
+  }
+
+  async isRedirectPage() {
+    const url = this.page.url();
+
+    const isCorrectPage =
+      url.startsWith("http://example.net") &&
```
[the remark this answers is not in the document]
Comment (merlinc) [reply tests/browser/pages/relying-party.js]: Replaced this with a proper URL parsing approach

File: README.md [hunk 2]
```diff
@@ -1,3 +1,134 @@
 # di-ipv-cri-check-hmrc-front
 
-HMRC Check Credential Issuer
+[![Github Action: Unit Tests](https://github.com/alphagov/di-ipv-cri-check-hmrc-front/actions/workflows/unit-tests.yml/badge.svg)](https://github.com/alphagov/di-ipv-cri-check-hmrc-front/actions/workflows/unit-tests.yml?query=branch%3Amain)
+[![Coverage](https://sonarcloud.io/api/project_badges/measure?project=alphagov_di-ipv-cri-kbv-front&metric=coverage)](https://sonarcloud.io/summary/new_code?id=alphagov_di-ipv-cri-check-hmrc-front)
+
+> Note: This repository is templated as part of [di-ipv-cri-templates](https://github.com/alphagov/di-ipv-cri-templates), and any substational changes that can be shared should be included back into that repository for re-use elsewhere
+
+Check Hmrc Credential Issuer Frontend is a Credential Issuer as part of the GOV.UK One Login programme.
```
Comment (scronk1) [inline README.md:8]: Should be `HMRC` not `Hmrc`

File: package.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "di-ipv-cri-check-hmrc-front",
+  "version": "1.0.0",
+  "description": "Frontend for the Check Hmrc Credential Issuer",
```
Comment (scronk1) [inline package.json:4]: _Minor_
`HMRC` not `Hmrc`

File: src/assets/javascripts/application.js
Comment (scronk1) [inline src/assets/javascripts/application.js:1]: Is this intentionally empty? If so is it necessary?

File: tests/browser/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# di-ipv-cri-check-hmrc-front-tests
+
+Check Hmrc Credential Issuer Frontend Tests is a test suite designed to run against the webserver in the root of this repository.
```
Comment (scronk1) [inline tests/browser/README.md:3]: `HMRC` not `Hmrc`

File: tests/browser/package.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "di-ipv-cri-check-hmrc-front-tests",
+  "version": "1.0.0",
+  "description": "Browser tests for the frontend of the Check Hmrc Credential Issuer",
```
Comment (scronk1) [inline tests/browser/package.json:4]: Minor
`HMRC` not `Hmrc`

**Pull request merged**